client/src/components/Header.tsx

```diff
@@ -0,0 +1,115 @@
+import AppBar from "@mui/material/AppBar";
+import Link from "@mui/material/Link";
+import InputBase from "@mui/material/InputBase";
+import SearchIcon from "@mui/icons-material/Search";
+import { styled, alpha } from "@mui/material/styles";
+import Toolbar from "@mui/material/Toolbar";
+import MenuItem from "@mui/material/MenuItem/MenuItem";
+import Menu from "@mui/material/Menu";
+import React from "react";
+import AccountCircle from "@mui/icons-material/AccountCircle";
+import IconButton from "@mui/material/IconButton/IconButton";
+
+const Search = styled("div")(({ theme }) => ({
+  position: "relative",
+  borderRadius: theme.shape.borderRadius,
```

Comment:
Should this be a theme value?? Border radius is:
1) component specific
2) not something that would change with a "theme"